src/kOS/AddOns/KerbalAlarmClock/KACAlarmWrapper.cs

```diff
@@ -30,14 +31,11 @@ private void InitializeSuffixes()
             AddSuffix("ID", new Suffix<string>(() => alarm.ID));
             AddSuffix("NAME", new SetSuffix<string>(() => alarm.Name, value => alarm.Name = value));
 
-            //AddSuffix("VESSELID", new SetSuffix<string>(() => alarm.VesselID, value => alarm.VesselID = value));
-            AddSuffix("VESSEL", new SetSuffix<VesselTarget> (() => getVesselByID (alarm.VesselID), setVesselID));

```

Comment:
I think that we should only show the alarms that are general or for the vessel requesting the list. if we do that we do not need to list the vessel at all.